keps/sig-network/2079-network-policy-port-range/README.md

```diff
@@ -232,20 +232,40 @@ _This section must be completed when targeting alpha to a release._
 
 * **Can the feature be disabled once it has been enabled (i.e. can we roll back
   the enablement)?**
-  Yes, but CNIs relying on the new field wont recognize it anymore
+  Yes, but CNIs relying on the new field wont recognize it anymore. Also, 
```

Comment:
It wouldn't recognize the field only if we rollback the Kubernetes version to version that doesn't have this field.

The question is purely about disabling the enablement, in which case the field is still around (though as you mentioned below it will get cleared on the next update).

So let's descibe those things separately:
- feature disablement (and the consequence of field being removed on next update)
- API disablement

Reply:
Did some changes, in the same item separated what happens when you disable the FG and what happens if you rollback to a K8s version that does not have this field yet.